README.md

```diff
@@ -1,2 +1,71 @@
 # commercetools-email-retry-processor
-Scheduled service which allows to send emails asynchronously and retry on errors.
+
+ The email retry processor fetches all emailobjects, which are saved in customobjects,  of the given tenants. For each
+ of this emailobjects, the processor checks, if they are in state "pending" and ,if true, it triggers a api endpoint.
+ This api endpoint should contain the email delivery logic.
+
+[![Build Status](https://travis-ci.org/commercetools/commercetools-email-retry-processor.svg?branch=create_cronjob)](https://travis-ci.org/commercetools/commercetools-email-retry-processor)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Prerequisites
+
+ - Java 8
+ - CTP project, for which the email should be delivered
```

Comment:
This [comment](https://github.com/commercetools/commercetools-email-retry-processor/pull/1#discussion_r152622027) wasn't addressed. This line is confusing because how can a CTP project receive an e-mail.

Reply:
647ce4b